crates/ln-dlc-node/src/node.rs

```diff
@@ -679,9 +681,107 @@ impl Node {
             .insert(intercept_scid, target_node);
         intercept_scid
     }
+
+    pub async fn initiate_dlc_channel(
```

Comment:
Should we rename this to something like `propose_dlc_channel`? If I understand it correctly then the other party needs to manually accept this step.

Reply:
Yep!